src/hooks/useEffectAfterMount.ts

```diff
@@ -0,0 +1,14 @@
+import { EffectCallback, useEffect, useRef } from "react";
+
+export default function useEffectAfterMount(fn: EffectCallback, inputs: ReadonlyArray<any>): void {
+  const hasMounted = useRef(false);
+
+  useEffect(() => {
+    if (hasMounted.current) {
+      return fn();
+    }
+    hasMounted.current = true;
+
+    return () => {};
+  }, [inputs]);
```

Comment:
Does it throw an error if you remove `return () => {};`

Reply:
Yep. ESlint throws:

"Expected to return a value at the end of arrow function"